projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -1291,6 +1547,1833 @@
   "advancements.quark.foxhound_speed_up_furnace.description": "让猎狐犬在熔炉上睡觉来加快熔炉的速度",
   "advancements.quark.shiba_help.title": "给小镇上色",
   "advancements.quark.shiba_help.description": "手持火把，在柴犬的帮助下清理一个暗处",
-  "advancements.quark.posion_baby.title": "青春永驻",
-  "advancements.quark.posion_baby.description": "用毒马铃薯喂食一只幼年动物，使其无法成长"
+  "advancements.quark.poison_baby.title": "青春永驻",
+  "advancements.quark.poison_baby.description": "用毒马铃薯喂食一只幼年动物，使其无法成长",
+  "advancements.quark.crab_in_a_bucket.title": "螃蟹捕获",
+  "advancements.quark.crab_in_a_bucket.description": "用桶捕获一只螃蟹",
+
+
+
+  "_comment": "配置翻译，置于文件末尾，便于未来比对更新非配置部分。",
+  "quark.config.general.q_button_on_the_right.name": "「q」按钮位于右侧",
+
+  "quark.config.general.advancement_visibility_depth.name": "进度可见深度",
+  "quark.config.general.advancement_visibility_depth.desc": "你能在进度界面中看到多深的进度。原版为2。\n允许的值：(0,)",
+
+  "quark.config.general.allowed_screens.name": "允许的界面",
```

```suggestion
  "quark.config.general.allowed_screens.name": "可显示按钮的界面",
```
原文"Allowed Srceens"，还是别翻译得这么直球了